salt/utils/saltminionservice.py

```diff
@@ -15,7 +15,7 @@
 import winerror
```

Comment:
I don't think this and `winservice.py` are even being used anymore. I think they were early attempts to create a Windows Service for the Salt-Minion. We should probably look at getting rid of them.